libs/notebook/ext.go

```diff
@@ -2,22 +2,42 @@ package notebook
 
 import "github.com/databricks/databricks-sdk-go/service/workspace"
 
-func GetExtensionByLanguage(objectInfo *workspace.ObjectInfo) string {
+type Extension string
```

Comment:
Why a type if there aren't any constructor or member functions?

I see the code in the filer casts any string to this type to make it work, so the value of this approach over a set of string constants is not clear to me.

Reply:
Good point, removed the `Extension` type.

Reply:
removed